extensionsGallery-insider.json

```diff
@@ -2398,7 +2398,7 @@
 					},
 					"versions": [
 						{
-							"version": "0.9.7",
+							"version": "0.9.8",
 							"lastUpdated": "3/3/2020",
```

Comment:
Update the "lastUpdated" fields too

Reply:
Good catch. Not sure how I missed that. Done. Thanks.